server/index.js

```diff
@@ -91,6 +96,35 @@ class DevtoolServer {
 
 }
 
+/**
+ * 
+ * Validate credentials in .env file 
+ */
+function validateCredentials() {
+    if (!((process.env.AZURE_STORAGE_ACCOUNT && process.env.AZURE_STORAGE_KEY && process.env.AZURE_STORAGE_CONTAINER_NAME) || 
+    (process.env.S3_BUCKET && process.env.AWS_ACCESS_KEY_ID && process.env.AWS_SECRET_ACCESS_KEY))
+    ) {
+        console.error('Error: Missing some or all cloud storage credentials.');
```

Comment:
I opted for `console.error` instead of throwing errors because throwing errors cause unhandled promises (even if thrown in the `run` function):
```
node:61270) UnhandledPromiseRejectionWarning: Error: Missing some or all cloud storage credentials.
    at validateCredentials (/Users/jamiedelbick/work/adobe/asset-compute-devtool/server/index.js:105:15)
    at DevtoolServer.run (/Users/jamiedelbick/work/adobe/asset-compute-devtool/server/index.js:39:9)
    at Object.start (/Users/jamiedelbick/work/adobe/asset-compute-devtool/server/index.js:125:19)
    at Object.<anonymous> (/Users/jamiedelbick/work/adobe/asset-compute-devtool/server/node_modules/run-func/index.js:23:28)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47
(node:61270) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:61270) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Reply:
Do we need to consider making this function async? It's called in an async function, after all.

Reply:
Yeah it can be. I don't have a strong opinion about it

Reply:
General comment: If the function doesn't do any truly async work, then we don't have to make it async (it would make it harder to use it somewhere else that is not async already).

In this particular case, it reads some files, which could also be done async. However IIUC here in the context of the startup of this cli app/command, the user experience must be blocking anyway (only continue after successful validation), the file access should be fast (all local files) and there are no concurrent tasks. Hence sync is fine IMO.